html/js/tree/Player.js

```diff
@@ -0,0 +1,65 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Player.
+    * @class
+    * @param {String} name Player's name.
+    */
+    function Player(id, colour) {
+        this.id = id;
+        if (this.id === 0) {
```

Comment:
Can you create a constant Player.CHANCE, with value of 0, and use it instead of 0? I think it makes the code more readable.

If you do so, please update too the following:
node .js lines 41,73,125,126,
tree.js line 213
(And double check I didnt miss any other, I think I didn't)